sat/cli/bootprep/validate.py

```diff
@@ -116,12 +116,17 @@ def validate_instance_schema_version(instance, schema_validator):
 
     # No default is needed here because the version property was added to the
     # bootprep_schema.yaml schema file simultaneously with this code.
-    current_version = schema_validator.schema['version']
-    strict_current_version = StrictVersion(current_version)
+    current_version = version.parse(schema_validator.schema['version'])
 
     try:
-        strict_instance_version = StrictVersion(instance_version)
-    except ValueError as err:
+        instance_version = version.parse(instance_version)
```

Comment:
This is pretty minor, but the reason I kept `instance_version` and `strict_instance_version` as separate vars was so that the exact string specified under the `schema_version` property in the input instance would be printed to the user in `err_template`. I think once it's parsed, the string representation can look a little different (e.g. zero padding removed).

Reply:
That's a good point. I can keep the instance version string around and use that in error messages instead.